spec/davinci_dtr_test_kit/dtr_light_ehr_user_response_test_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'erb'
+require_relative '../../lib/davinci_dtr_test_kit/tags'
+
+RSpec.describe DaVinciDTRTestKit::DTRLightEHRUserResponseTest, :request do
+  let(:test) { described_class }
+  let(:suite_id) { :dtr_light_ehr }
+  let(:unique_url_id) { '12345' }
+  let(:supported_payer_url) { "/custom/#{suite_id}/#{unique_url_id}/supported-payers" }
+  let(:results_repo) { Inferno::Repositories::Results.new }
+  let(:test_session) { repo_create(:test_session, test_suite_id: suite_id) }
+  let(:result) { repo_create(:result, test_session_id: test_session.id) }
+
+  def create_supported_payers_request(user_response = nil)
+    headers = [
+      {
+        type: 'request',
+        name: 'Accept',
+        value: 'application/json'
+      }
+    ]
+    url = supported_payer_url
+    url += "?user_response=#{ERB::Util.url_encode(user_response)}" if user_response.present?
```

Comment:
This is the wrong place for this to go - should be in the response body. I think this was something you did previously, but need to update.